engine/src/main/scala/cromwell/webservice/ApiDataModels.scala

```diff
@@ -36,7 +36,7 @@ case class WorkflowMetadataResponse(id: String,
                                     calls: Map[String, Seq[CallMetadata]],
                                     failures: Option[Seq[FailureEventEntry]])
 
-case class WorkflowQueryResponse(results: Seq[WorkflowQueryResult])
+case class WorkflowQueryResponse(results: Seq[WorkflowQueryResult], page: Option[Int], pageSize: Option[Int], totalRecords: Int)
```

Comment:
yeah query workflows always gets the count. Are you suggesting that we only run the count query if we are requesting pagination?